test/ulid/rails_test.rb

```diff
@@ -44,6 +44,12 @@ def test_validate_ulid_format
     assert_raises(ArgumentError) { User.new(id: non_ulid_id) }
   end
 
+  def test_should_save_when_ulid_foreign_key_is_nil
```

Comment:
Avoid using "should" when describing your tests, express it with an action instead.

"should do this" -> "does this"

```suggestion
  def test_saves_when_ulid_foreign_key_is_nil
```